skin/adminhtml/default/default/mm_monacoeditor.css

```diff
@@ -0,0 +1,3 @@
+.hor-scroll {
+    overflow: initial;
+}
```

Comment:
What problem does this solve? I have tried it with the default theme and I did not encounter any issues even without this CSS.

Reply:
Without this fix it may look like this:
<img width="561" alt="Bildschirmfoto 2023-11-23 um 11 14 15" src="https://github.com/empiricompany/openmage-mm_monacoeditor/assets/26252058/61272478-dd96-45ae-a20d-410740e66acb">